Dfe.Academies.Academisation.Data/Repositories/TransferProjectRepository.cs

```diff
@@ -19,6 +19,17 @@ public TransferProjectRepository(AcademisationContext context) : base(context)
 		{
 			return await DefaultIncludes().SingleOrDefaultAsync(x => x.Urn == urn);
 		}
+
+		public async Task<IEnumerable<ITransferProject>> GetTransferProjectsByIncomingTrustUkprn(string ukprn, CancellationToken cancellationToken)
```

Comment:
this method is filtering by more than the ukprn, the group id and status are also filtered upon it would be better to have this be a specific method to get transfer projects for group or do the filtering of group and status in the query if performance is not a factor.

Reply:
Updated the name to `GetTransfersProjectsForGroup` specific endpoint for returning transfer projects for group